technical-reports/format/design-token.md

```diff
@@ -90,20 +90,20 @@ Please raise concerns if these limitations create problems for implementers.
 
 ## Additional properties
 
-While "value" is the only required property for a token, a number of additional properties may be added:
+While "value" is the only required property for a token, a number of additional properties MAY be added:
 
 ## Description
 
 A plain text description explaining the token's purpose. Tools MAY use the description in various ways.
 
 For example:
 
-- Style guide generators could display the description text alongside a visual preview of the token
```

Comment:
I agree. This feels like an overreach of a find/replace.

Reply:
Actually, looking at this more closely, I think the MAY's make sense here. Yes, they are examples, but they're describing potential tool functionality and using MAY helps with interoperability since it's not required functionality. I may be reading this wrong though.

Reply:
Fair point. I'm inclined to agree with your @kevinmpowell 

FWIW, reviewing this PR has made me realise applying RFC 2119 appropriately is a lot harder than it looks!